### DISCUSSION TITLE
Github page slow to load

### DISCUSSION THREAD

**Discussion by FahdSeddik** [community/community#33752] [Other Feature Feedback, Questions, & Ideas]
Hey!
I had my website hosted on GitHub at [fahdseddik.github.io](https://fahdseddik.github.io/). The website seems to take a long time to load for new users who have not loaded it before. Does anyone have a fix for this? I tried to see what was slowing it down by removing unnecessary CSS files or converting image format or using 'defer' tags etc. but it did not work. I would be glad if someone can help. Thanks!

**Comment by samuel-ma** [comment]
Oooo @FahdSeddik 
Is this some sort of humble flex or what my G
That's a great looking portfolio you have there
Who designed that?

Also, there's a "come back" at the header which grabs my attention each time I leave, bravo...
Looks like you've really put in some effort in making it
That's 5 stars from me

Anyways, took too long to load tho, I wonder why.....
Maybe cause of all the particles running in the background
There isn't much Html on your site, so technically, it should be ok

Idk if I can help much, but try searching for a free tool to test and monitor your page's performance
Try this [here.......](https://tools.pingdom.com/#60d88e5177c00000)

**Comment by jfullstackdev** [comment]
I don't think so

![image](https://user-images.githubusercontent.com/47092464/191670323-52578d49-b9e4-445f-9a15-f6b908ceec40.png)

it depends on the computer or user end maybe, it loaded fast on my end,

if you are really concerned, you can separate your images, videos and any other sources using CDN, so it will be extremely fast
